woocommerce/woocommerce-android
# [Improvement] Display Applied Coupons in Order Summary

Closes: #12619


### Description


This PR addresses the issue of applied coupons not being displayed in the order summary section and only appearing in the totals drawer at the bottom. The applied coupons will now be shown in the same area as the "Add Coupon" button, similar to how shipping details are displayed. 


### Steps to reproduce


1. Create a new order and add some products.
2. Apply a coupon using the "Add Coupon" button.
3. Add more coupons
4. Observe that the coupon is now displayed directly under the "Add Coupon" button in the order summary, as well as in the totals drawer at the bottom.

### Testing information


#### Devices Tested
- emulated phone
- emulated tablet


#### Testing Scenarios
- Apply single coupon and verify display.
- Apply multiple coupons and ensure all are displayed correctly.
- Remove a coupon and check that it disappears from the order summary.




### The tests that have been performed


- Verified the display of applied coupons in the order summary section.
- Tested the application and removal of coupons.

  
### Images/gif


**Before:**


<img src="https://github.com/user-attachments/assets/873029d2-e21d-49e5-bf7b-5318e99cf08b" alt="shipping_row_example"  height="420">  



**After:**


TBD


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
